### PR TITLE
[3211] Add Japanese translation

### PR DESCRIPTION
**Pull request by reika727**
I translated SCP-3211 into Japanese. Thank you for your great work!

### CODE REVIEW

File: articles/scp-3211/src/jp/loading.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="warning-middle">
+  <p><b>SCiPnet</b></p>
+  <p>Loading file</p>
+</div>
```
Comment (rossjrw) [inline articles/scp-3211/src/jp/loading.html:4]: ^

File: articles/scp-3211/src/jp/warning.ejs.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<div class="warning-container">
+  <div class="warning-image">
+    <p>INFOHAZARD<br>WARNING</p>
```
Comment (rossjrw) [inline articles/scp-3211/src/jp/warning.ejs.html:3]: ^

File: articles/scp-3211/src/jp/warning.ejs.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <p>
+      この文書を読み進めることは、異常効果へ暴露することを理解・受容したことを意味します。
+    </p>
+  </div>
+  <div class="warning-middle">
+    <p>
+      <b>
+        LEVEL 4/3211
+        <span id="authorise">AUTHORISATION</span>
+        REQUIRED
```
Comment (rossjrw) [inline articles/scp-3211/src/jp/warning.ejs.html:25]: ^

**Review by rossjrw — APPROVED** [review]
Perfect - thank you so much!

In case you weren't able to generate it locally, the Wikitext copy-paste for JP is here: https://gist.github.com/rossjrw/910fb203037ccbae7474e79a1fe49e69

I've noted a couple of places that have untranslated text - but if you intentionally chose not to translate these, please ignore these suggestions! (If that's the case, leave a comment to say so and I'll merge this PR)

**Comment by reika727** [comment]
Thank you for your approval. [Japanese Translation](http://scp-jp.wikidot.com/scp-3211) is now available!

Yes, those untranslated texts are intentional :thumbsup:

**Pull request merged**